server/routes/validation/group.js

```diff
@@ -0,0 +1,38 @@
+import Joi from 'joi';
+
+export default {
+
+  createGroup: {
+    body: {
+      name: Joi.string().required(),
+      users: Joi.array().items(Joi.string().hex().length(24)),
+      escalationPolicy: Joi.object().keys({
+        rotationInterval: Joi.number(),
+        pagingInterval: Joi.number(),
+        subscribers: Joi.array().items(Joi.string().hex().length(24))
+      })
+    }
+  },
+
+  byName: {
+    params: {
+      groupName: Joi.string().required()
+    }
+  },
+
+  updateGroup: {
+    options: { allowUnknownBody: false },
```

Comment:
Do you want the other validations to fail with invalid keys passed in with the object?